trillian/integration/ct_integration_test.go

```diff
@@ -45,10 +47,9 @@ var (
 	skipStats      = flag.Bool("skip_stats", false, "Skip checks of expected log statistics")
 )
 
-func TestLiveCTIntegration(t *testing.T) {
-	flag.Parse()
+func commonSetup() ([]*configpb.LogConfig, error) {
```

Comment:
You could pass in t here, and then call `t.Helper()`
Save you have to return err and check it everywhere.

Reply:
Yeah I broke it as it used to call t.Skip()

Reply:
Done.